domain-models-runtime/src/main/java/org/folio/rest/persist/PgUtil.java

```diff
@@ -554,11 +554,19 @@ private static void streamTrailer(HttpServerResponse response, ResultInfo result
     String element, RoutingContext routingContext, Map<String, String> okapiHeaders,
     Context vertxContext) {
 
+   streamGet(table,clazz,cql,offset,limit,facets,element,routingContext,okapiHeaders,vertxContext,0);
+  }
+
+   public static <T> void streamGet(String table, Class<T> clazz,
+    String cql, int offset, int limit, List<String> facets,
+    String element, RoutingContext routingContext, Map<String, String> okapiHeaders,
+    Context vertxContext, int limitRequestExecutionTime) {
```

Comment:
The variable names `limitRequestExecutionTime`, `executionTimeLimit`, and `EXECUTION_TIME_LIMIT` can be replaced by the much shorter `timeout` and `TIMEOUT` without any information loss. Dropping name variants improves consistency.